src/settings/_font.scss

```diff
@@ -6,29 +6,35 @@ $ff-sans-serif: 'Open Sans', Helvetica, sans-serif !default;
 // Font weights
 $fw-light: 300 !default;
 $fw-regular: 400 !default;
-$fw-semi-bold: 600 !default;
+$fw-semi-bold: 500 !default;
 $fw-bold: 700 !default;
 
 // Font sizes
-$fz-base: 14 !default;
-$fz-body: $fz-base * 1px !default;
-$fz-xs: ceil($fz-base * .666) * 1px !default;
-$fz-s: ceil($fz-base * .8) * 1px !default;
-$fz-m: $fz-base * 1px !default;
-$fz-l: ceil($fz-base * 1.25) * 1px !default;
-$fz-xl: ceil($fz-base * 1.75) * 1px !default;
+$fz-m: 16px !default;           // 16px
```

Comment:
I don't get this...
I think that font sizes should be proportional to the font-base as it was previously defined

Reply:
Font-base is not a concept of design. We can add it later when we actually remove the compat_v7 layer in every project.

Reply:
agree with @davecarter. i would define a `fz-base` rather than use `$fz-m`. but this is a minor comment.

Reply:
@JoanClaret @davecarter done !